delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/permissions/StoragePermissionProvider.scala

```diff
@@ -0,0 +1,12 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.permissions
+
+import ch.epfl.bluebrain.nexus.delta.sdk.model.IdSegmentRef
+import ch.epfl.bluebrain.nexus.delta.sdk.permissions.model.Permission
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.ProjectRef
+import monix.bio.UIO
+
+trait StoragePermissionProvider {
```

Comment:
Is this a sensible interface? It's used because in the module where the permissions check happens, we don't have visibility of the storage plugin

Reply:
It looks ok to me. Later it could be made more general to just PermissionProvider no (if needed)?